docs/content/concepts/automation/schedules/testing.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: "Testing schedules | Dagster Docs"
+description: "Learn to test your schedules in Python or the Dagster UI."
+---
+
+# Testing schedules
+
+In this guide, we'll show you how to use the Dagster UI and Python to test your [schedules](/concepts/partitions-schedules-sensors/schedules). For more information about testing in Dagster, refer to the [Testing documentation](/concepts/testing).
+
+---
+
+## Testing schedules in the Dagster UI
+
+Using the UI, you can manually trigger test evaluations of a schedule and view the results. This can be helpful when [creating a schedule](/concepts/automation/schedules/automating-assets-schedules-jobs) or for [troubleshooting unexpected scheduling behavior](/concepts/automation/schedules/troubleshooting).
+
+1. In the UI, click **Overview > Schedules tab**.
+
+2. Click the schedule you want to test.
+
+3. Click the **Test Schedule** button, located near the top right corner of the page.
+
+4. You'll be prompted to select a mock schedule evaluation time. As schedules are defined on a cadence, the evaluation times in the dropdown are past and future times along that cadence.
```

Comment:
Worth calling out why you'd pick a certain evaluation time over others?

ie. 

`This is helpful when you have a daily schedule that skips national holidays."